tests/components/grid.test.ts

```diff
@@ -1427,6 +1427,20 @@ describe("Copy paste keyboard shortcut", () => {
     expect(getCellContent(model, "A2")).toEqual("things");
   });
 
+  test("cut zone gets cleared on paste if content/style is altered after cut", async () => {
+    setCellContent(model, "A1", "things");
+    setStyle(model, "A1", { bold: true });
+    selectCell(model, "A1");
+    document.body.dispatchEvent(getClipboardEvent("cut", clipboardData));
+    setCellContent(model, "A1", "new content");
+    setStyle(model, "A1", { bold: false });
+    selectCell(model, "A2");
+    document.body.dispatchEvent(getClipboardEvent("paste", clipboardData));
+    expect(getCellContent(model, "A2")).toEqual("things");
```

Comment:
A2 should contain "new content" no (for CUT) ? Here the changes we made to A1 are totally lost

Reply:
If we go according to the specs of this task, which says PASTE the old values & styles made before CUT/COPY, then I guess this test does solve the purpose right?. Also, this behaviour is same as it was before this fix/task. So do you intend to keep the new values after cut instead? Or keep the old ones as it was?

Reply:
When we copy, we should put things in the clipboard, and then paste it as it was copied, without worrying about the changes in the sheet.

Cut/paste is a bit trickier since we delete some sheet content, and we don't want to lose information. Cut can be seen as moving cells from one place to another, so we should use the current value of the sheet.

That's the theory anyway, but looking at the current code we never really did that. I'd say you can leave it like that, and we'll do another task to handle cut correctly.

Reply:
Okay, I'll leave it as it is then.

Suggestion: We can try to replicate the cut/paste behaviour as in gsheet in the new task.